CONTRIBUTING.md

```diff
@@ -59,3 +61,46 @@ It allows us to do following when saving a file:
 * Remove unnecessary semicolon
 * Add missing @Override annotations
 
+## Coding conventions
```

Comment:
Do we name conventions?

Reply:
It's covered in the referenced Google style guide.